lib/src/layer/polyline_layer.dart

```diff
@@ -69,28 +69,146 @@ class PolylineLayer extends StatelessWidget {
   const PolylineLayer({
     super.key,
     required this.polylines,
-    this.polylineCulling = false,
+    this.polylineCulling = true,
+    this.polylineCullingMargin = 0,
     this.simplificationTolerance = 1,
     this.simplificationHighQuality = false,
   });
 
+  bool aabbContainsLine(
```

Comment:
Should this be public?

Reply:
it could be. i might move it into the bounding box class as it is probably more appropriate there.

Reply:
Sure. I was actually asking whether it would make sense for it to be private, it just seemed like you may have made a mistake, as it's quite an "out of the way" place to put it. But if you're moving it somewhere better suited it's probably not so much of an issue.